tests/bsim/bluetooth/host/adv/periodic/tests_scripts/per_adv_app_not_scanning.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Basic periodic advertising sync test: an advertiser advertises with periodic
```

Comment:
This seems just copy pasted from the previous test.
Would it make sense to modify it to describe a bit better this particular test? (same for the next one)